src/litserve/api.py

```diff
@@ -76,6 +76,8 @@ def _unbatch_stream(self, output_stream):
 
     def unbatch(self, output):
         """Convert a batched output to a list of outputs."""
+        if self._default_unbatch is None:
+            raise ValueError("Unbatch function is not set.")
```

Comment:
Let's make the error more user friendly, something like - "Default implementation for `LitAPI.unbatch` method was not found, please implement LitAPI.unbatch method."

Reply:
Sure done. Also added a test case for the `ValueError`.